duckduckgo/Android
# Remove user agent unless on Wikipedia

Task/Issue URL: https://app.asana.com/0/inbox/1125189844075764/1200283621127968/1200292484141654
Tech Design URL: 
CC: 

**Description**:
Akamai is blocking our user agent so lots of sites are failing. This PR only adds the DuckDuckGo bit for SERP and wikipedia.

**Steps to test this PR**:
1. Go to `homedepot.com` and the website should load.
1. Go to `accuweather.com` and the website should load.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
